pkg/server/chaosd/jvm.go

```diff
@@ -0,0 +1,214 @@
+// Copyright 2020 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package chaosd
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"text/template"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
+
+	"github.com/chaos-mesh/chaosd/pkg/core"
+)
+
+const ruleTemplate = `
+RULE {{.Name}}
+CLASS {{.Class}}
+METHOD {{.Method}}
+AT ENTRY
+IF true
+DO 
+	{{.Do}};
+ENDRULE
+`
+
+const stressRuleTemplate = `
+RULE {{.Name}}
+STRESS {{.StressType}}
+{{.StressValueName}} {{.StressValue}}
+ENDRULE
+`
+
+const gcRuleTemplate = `
+RULE {{.Name}}
+GC
+ENDRULE
+`
+
+type jvmAttack struct{}
+
+var JVMAttack AttackType = jvmAttack{}
+
+const bmInstallCommand = "bminstall.sh -b -Dorg.jboss.byteman.transform.all -Dorg.jboss.byteman.verbose -p %d %d"
```

Comment:
I suggest to give way to install byteman in chaosd , or auto install it or support use path of file such as 'bminstall.sh'.Anyway,I think code in chaosd dependent on configure in other repo is not a good idea.
`bminstall.sh` here rely on ```export BYTEMAN_HOME=`pwd`/byteman-download-4.0.14
export PATH=$PATH:${BYTEMAN_HOME}/bin``` in byteman setting.

Reply:
I think we can involve all the tools or libs required for chaosd in it's download package file. The package may looks like:

chaosd-linux-v0.9.0
     |── bin
     │   └── chaosd
     |── tools
         |── byteman
         |── stress-ng

Reply:
@cwen0 Do you think it's OK?

Reply:
Can we move out `chaosd` binary from `bin` directory?

Reply:
It look good to me